src/sv-pipeline/scripts/make_evidence_qc_table.py

```diff
@@ -0,0 +1,264 @@
+"""
+This script creates a single table of sample QC & batching metrics from EvidenceQC.
+"""
+
+import argparse
+import pandas as pd
+from collections import Counter
+from numpy import array
+from functools import reduce
+
+
+ID_COL = "#ID"
+
+
+def read_ploidy(filename: str) -> pd.DataFrame:
+    """
+
+    Args:
+        filename: A tab-delimited file containing estimated copy numbers.
+
+    Returns:
+        A pandas DataFrame containing the following columns:
+        [id, chr1_CopyNumber, ..., chr22_CopyNumber, chrX_CopyNumber, chrY_CopyNumber, chrX_CopyNumber_rounded].
+    """
+    df_ploidy = pd.read_csv(filename, sep="\t")
+    df_ploidy.loc[round(df_ploidy["chrX_CopyNumber"]) < 2, "chrX_CopyNumber_rounded"] = 1
+    df_ploidy.loc[round(df_ploidy["chrX_CopyNumber"]) >= 2, "chrX_CopyNumber_rounded"] = 2
+    return df_ploidy
+
+
+def read_bincov_median(filename: str) -> pd.DataFrame:
+    """
+    Median coverage (bincov_median)
+    Args:
+        filename: bincov_median output from EvidenceQC (medianCov.transposed) in a tab separated format
+
+    Returns:
+        a pandas DataFrame containing the transposed table given in the input file.
+    """
+    df_median = pd.read_csv(filename, sep="\t").T
+    df_median = df_median.rename_axis(ID_COL).reset_index()
+    df_median.columns = [ID_COL, "median_coverage"]
+    df_median = df_median.reset_index(drop=True)
+    return df_median
+
+
+def read_wgd_scores(filename: str) -> pd.DataFrame:
+    return pd.read_csv(filename, sep="\t")
```

Comment:
Can you name the output column `wgd_score`? Currently it is named `score` in the original WGD file, but that is ambiguous in the unified output table

Reply:
Renamed to `wgd_score`.